reporter/src/funTest/kotlin/reporters/ExcelReporterTest.kt

```diff
@@ -38,6 +39,20 @@ class ExcelReporterTest : WordSpec({
             val outputDir = createTempDir().apply { deleteOnExit() }
             ExcelReporter().generateReport(ortResult, DefaultResolutionProvider(), CopyrightGarbage(), outputDir)
             outputDir.resolve("scan-report.xlsx").isFile shouldBe true
+
+            val actualFile = outputDir.resolve("scan-report.xlsx")
+            val actualXlsxUnpacked = createTempDir().apply { deleteOnExit() }
+            actualFile.unpackZip(actualXlsxUnpacked)
+
+            val expectedFile = File("src/funTest/assets/file-counter-expected-scan-report.xlsx")
```

Comment:
Would it make sense to commit two separate expected files for sheet1 and sheet2 rather than one archive file?